spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/AppDeployer.java

```diff
@@ -56,6 +56,12 @@
 	static final String INDEXED_PROPERTY_KEY = PREFIX + "indexed";
 
 	/**
+	 * Constant property key to be used in the deployer implementation as an environment variable
```

Comment:
I wouldn't say "Constant property key" here, because it is confusing given that other constants are named `*_PROPERTY_KEY` (and play a very different role).
I'll change this part on merge